packages/cli/src/Init.ts

```diff
@@ -18,10 +18,15 @@ import { bold, dim, green, red, yellow } from 'kleur/colors'
 import path from 'path'
 import { match, P } from 'ts-pattern'
 
+import { poll } from './platform/_'
+import { credentialsFile } from './platform/_lib/credentials'
+import { successMessage } from './platform/_lib/messages'
 import { printError } from './utils/prompt/utils/print'
 
+type ConnectorTypeOrPrismaPostgres = ConnectorType | 'prismapostgres'
```

Comment:
I tried adding the `prismapostgres` to ConnectorType definition [here](https://github.com/prisma/prisma/blob/a97413b1e14c24190cf85d2fd3fe2b3953ebc3de/packages/generator-helper/src/types.ts#L66-L73).

This produced a bunch of other type errors. I was hesitant to add `prismapostgres` through out the code base.

Should we? Or is this temporary local type fine?